Equipment/EquipmentDb/Misc/trinkets.xml

```diff
@@ -174,6 +174,21 @@
 			"Blessed by the Shen'dralar Ancients."
 		</flavour_text>
 	</item>
+	
+	<item id="18465" phase="2">
+		<info name="Royal Seal of Eldre'Thalas" type="TRINKET" slot="TRINKET" unique="yes" req_lvl="54" item_lvl="62" quality="RARE" boe="no" icon="Inv_jewelry_talisman_09.png"/>
```

Comment:
Should be Inv_jewelry_talisman_10.png for the correct image, no? It is available for other classes' seals so only need to change this line, no need to add the icon.

Reply:
Ah, yes, you're right. 09 and 10 look very similar if you don't have them side by side.